README.md

```diff
@@ -1,2 +1,45 @@
 # build-raven-os-org
 Work In Progress - Will be hosted at https://build.raven-os.org
+
+## Installation
+
+```bash
+$ apt-get install postgres postgresql-contrib
+$ update-rc.d postgresql enable
+$ service postgresql start
+$ apt-get install libpq-dev
+# connect to postgres and set a password using \password
+
+# set your configuration in src/config.local.js
```

Comment:
the example is `src/config/config.js`, the user should but instead of modifying this file, the user create his local config `src/config/config.local.js` which override the `src/config/config.js` file but is not pushed on the repository.